features/0334-jwe-envelope/README.md

```diff
@@ -166,26 +167,24 @@ In the above two examples, `apu` is the encoded ephemeral key used to encrypt th
 {
     "protected": base64url({
         "typ": "didcomm-envelope-enc",
-        "enc":"XC20P", // or "A256GCM"
+        "enc":"XC20P", // or one of: "A128CBC-HS256", "A192CBC-HS384", "A256CBC-HS512"
```

Comment:
```suggestion
        "enc":"A256CBC-HS512",
```

Since the alg is ECDH-1PU don't we only have the option to use a CBC mode here from our options? I may be missing context since I'm only looking at the JSON and not the broader detail of what's being described around it.

Reply:
That's right, but only `"A256CBC-HS512"` is required, the other two are optional, I can probably add them back as a comment. Also I've removed XC20P since 1PU should not support it.

Reply:
Updated `enc` entry for 1PU to:
`"enc":"A256CBC-HS512", // or one of: "A128CBC-HS256", "A192CBC-HS384"`